### PR TITLE
EOS-16371 Auth:Openldap upgrading from openldap -> symas-openldap

### PR DESCRIPTION
**Pull request by sachitanands**
Upgrade openldap rpms .
Openldap -> symas-openldap 
version is 2.4.58

Signed-off-by: Sachitanand Shelake <sachitanand.shelake@seagate.com>

### CODE REVIEW

File: scripts/ldap/setup_ldap.sh [hunk 1]
```diff
@@ -94,8 +94,12 @@ rm -rf /etc/openldap/slapd.d/cn\=config/olcDatabase\=\{2\}mdb.ldif
 if [[ $forceclean == true ]]
 then
   rm -rf /var/lib/ldap/*
+  yum remove -y symas-openldap-clients symas-openldap-servers
 fi
 
+yum install -y symas-openldap-clients symas-openldap-servers
```
Comment (sachitanands) [inline scripts/ldap/setup_ldap.sh:100]: add symas-openldap rpm as well.

File: scripts/ldap/setup_ldap.sh [hunk 2]
```diff
@@ -94,8 +94,12 @@ rm -rf /etc/openldap/slapd.d/cn\=config/olcDatabase\=\{2\}mdb.ldif
 if [[ $forceclean == true ]]
 then
   rm -rf /var/lib/ldap/*
+  yum remove -y symas-openldap-clients symas-openldap-servers
```
Comment (sachitanands) [inline scripts/ldap/setup_ldap.sh:97]: for force clean , we dont remove pkg installation ..
we need to handle only data deletion

Comment (sachitanands) [reply scripts/ldap/setup_ldap.sh]: done

File: scripts/ldap/setup_ldap.sh (hunk 1 above)
Comment (sachitanands) [reply scripts/ldap/setup_ldap.sh]: not required because we removed this line

**Comment by sachitanands** [comment]
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Component-Test/job/S3Server-PreMerge-Test/1564/

**Review by nileshgovande — APPROVED** [review]

**Comment by sachitanands** [comment]
triggered R2 1 node deployment with S3, RE, Provisioner patch
http://eos-jenkins.mero.colo.seagate.com/job/Cortx-Deployment/job/VM-Deployment-R2-1Node/1177

**Review by bkirunge7 — APPROVED** [review]

**Pull request merged**